### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Poussinou**
Hi,

This small and useless PR adds a button to get your app on F-Droid. You can later add a button for Google Play the same way :
`<a href="" target="_blank"> Get it on Google Play</a>`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 A simple game inspired by Age of Castles. Distribute your population to farming, construction, guard duty and scholarship, survive demon invasion and find the way to banish them forever.
 
+<a href="https://f-droid.org/packages/hr.kravarscan.enchantedfortress/" target="_blank">
```
Comment (mueller-ma) [inline README.md]: I haven't seen other readme so far, that use `target="_blank"`, but otherwise LGTM.

Comment (Poussinou) [reply README.md]: I can remove it if you want?

Comment (mueller-ma) [reply README.md]: Yes, that would be good.

Comment (Poussinou) [reply README.md]: Done :)

**Comment by subchannel13** [comment]
Looks good, can go directly to master (usually PRs should go to dev branch). Thank you @Poussinou!

**Pull request merged**